src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -4489,7 +4489,7 @@ namespace VariableSpeedCoils {
         auto &ZoneEqSizing(state.dataSize->ZoneEqSizing);
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        Real64 rhoair;
+        Real64 rhoair = state.dataEnvrn->StdRhoAir;
```

Comment:
I'll put comments where code changes were made. 1) Use StdRhoAir, and 2) use zone temp as coil inlet T are the 2 changes in this PR